src/helpers/requester.js

```diff
@@ -0,0 +1,113 @@
+class Request {
```

Comment:
dont use / copy this from normal react projects. there is a built in mozilla library that cleanly handles requests called `fetch`, and you can make a simpler wrapper around that. we use it in https://github.com/calblueprint/replate-marketplace/tree/andrew/login/requesters

Reply:
Ye thanks! I was actually gonna do that as well. @nikitr I’ll put it in another pr before GM

Reply:
Perfect, thanks!